packages/compass-smoke-tests/src/installers/windows-setup.ts

```diff
@@ -72,7 +84,20 @@ export function installWindowsSetup({
   console.warn(
     "Installing globally, since we haven't discovered a way to specify an install path"
   );
-  execute(filepath, []);
+  execute(
+    filepath,
+    [
+      // Args are passed through to the Update.exe https://github.com/Squirrel/Squirrel.Windows/blob/51f5e2cb01add79280a53d51e8d0cfa20f8c9f9f/src/Setup/winmain.cpp#L125
+      // See options in https://github.com/Squirrel/Squirrel.Windows/blob/51f5e2cb01add79280a53d51e8d0cfa20f8c9f9f/src/Update/StartupOption.cs
+      '--silent',
```

Comment:
What does --silent do, though? That link just says `{ "s|silent", "Silent install", _ => silentInstall = true},`

Reply:
It'll avoid showing the install GUI and starting the application (which is what I think might be causing some issues).